uber/templates/preregistration/form.html

```diff
@@ -91,7 +95,7 @@
         <label for="tables" class="col-sm-2 control-label">Tables</label>
         <div class="col-sm-6">
             <select class="form-control" name="tables">
-                {% options c.TABLE_OPTS group.tables %}
+                {% int_options 1 4 group.tables %}
             </select>
```

Comment:
nice, this is consistent with what Danielle said, she wanted dealers to only be able to pick 4 tables.  on the backend though, she wants the ability to manually override this to add more tables.  I think it worked like this last year, and I'm guessing this change won't affect that.

Only minor nitpick is that we might want a c.MAX_SIGNUP_TABLES_ALLOWED or similar instead of hardcoding "4" in here.  Not worth holding up this PR though.